dev-mode/sbt-plugin/src/main/scala/sbt/PlayRun.scala

```diff
@@ -188,20 +195,26 @@ object PlayRun {
     @tailrec def shouldTerminate: Boolean =
       (System.in.available > 0) && (isEOF(System.in.read()) || shouldTerminate)
 
-    val sourcesFinder = () => {
-      watched.watchSources(state).iterator.flatMap(new PlaySource(_).getPaths).collect {
-        case p if Files.exists(p) => better.files.File(p)
-      }
+    val sourcesFinder: Supplier[java.lang.Iterable[java.io.File]] = () => {
+      watched
+        .watchSources(state)
+        .iterator
+        .flatMap(new PlaySource(_).getPaths)
+        .collect {
+          case p if Files.exists(p) => better.files.File(p).toJava
```

Comment:
```suggestion
          case p if Files.exists(p) => p.toFile
```
Now better-files is not necessary anymore and the dependency can be removed also